service/history/workflowExecutionContext.go

```diff
@@ -257,14 +257,21 @@ func (c *workflowExecutionContextImpl) loadWorkflowExecutionForReplication(
 		}
 
 		c.stats = response.State.ExecutionInfo.ExecutionStats
+		if c.stats == nil {
+			// TODO (alex): This check is for compatibility only and can be removed later
+			c.stats = &persistencespb.ExecutionStats{
+				HistorySize: response.State.ExecutionInfo.GetHistorySize(),
```

Comment:
This is usage of deprecated field. I don't know when and who removed it.

Reply:
It was me: https://github.com/temporalio/temporal/pull/883/files#diff-1a184a9b9694acc9af29fdc12ed1d4a68369c2eec5caa9e221cbae97fdfdcbf2